### ISSUE TITLE
tasks fails to bootstrap in v3.210410

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#6531]
AdjustSites.py fails with
```
2021-04-13 07:12:40.927765: Uploading webdir http://vocms059.cern.ch/mon/cms1627/210413_071205:belforte_crab_20210413_091158 to the REST
2021-04-13 07:12:41.049602: Webdir URL has been uploaded, exit code is 0. Setting the classad for the proxied webdir
Traceback (most recent call last):
  File "AdjustSites.py", line 489, in <module>
    main()
  File "AdjustSites.py", line 427, in main
    saveProxiedWebdir(RESTServer, ad)
  File "AdjustSites.py", line 303, in saveProxiedWebdir
    proxied_webDir = getProxiedWebDir(RESTServer=RESTServer, task=task, uriNoApi=uriNoApi, logFunction=printLog)
TypeError: getProxiedWebDir() got an unexpected keyword argument 'RESTServer'

```

**Comment by belforte** [comment]
this also happens in v3.210412

**Comment by belforte** [comment]
solved by #6536 together with  #6530
